test/restart/mzcompose.py

```diff
@@ -451,6 +451,76 @@ def workflow_allow_user_sessions(c: Composition) -> None:
     assert cursor.fetchall() == [(1,)]
 
 
+def workflow_network_policies(c: Composition) -> None:
```

Comment:
Thanks for adding a test! Double thanks for writing it as an mzcompose test and not an environmentd Rust test!